schema-registry/core/src/main/java/com/hortonworks/registries/schemaregistry/SchemaMetadataStorable.java

```diff
@@ -224,8 +230,17 @@ public static SchemaMetadataStorable updateSchemaMetadata(SchemaMetadataStorable
         schemaMetadataStorable.setSchemaGroup(schemaMetadata.getSchemaGroup());
         schemaMetadataStorable.setName(schemaMetadata.getName());
         schemaMetadataStorable.setDescription(schemaMetadata.getDescription());
-        schemaMetadataStorable.setCompatibility(schemaMetadata.getCompatibility());
-        schemaMetadataStorable.setValidationLevel(schemaMetadata.getValidationLevel());
+
+        schemaMetadataStorable.setCompatibility(
```

Comment:
When generated from jsons (which have null values)  with default constructor can  lead to nulls.